code/modules/mob/living/carbon/xenomorph/facehuggers.dm

```diff
@@ -451,7 +456,8 @@
 			return FALSE
 
 	if(on_fire)
-		return FALSE
+		if(fire_stacks >= 6)
```

Comment:
magic number bad.

Reply:
What do you mean and what do you want me to change about this?

Reply:
Magic number, as in it's just a number that's flying around just because. probably word it like "if on fire and fire_stacks greater than FLARE_FIRE_STACKS, Return FALSE"

Reply:
5 is a specific number of fire stacks applied by flares. You reference it elsewhere in this pr, but here you just have the number 6. What happens if flare firestacks changes in the future?

Reply:
Just make it a define, basically and have flares use it when deciding how many firestacks to apply and compare that define here instead of a flat number